src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx

```diff
@@ -918,16 +918,16 @@ export class Flyout extends Component<FlyoutProps, FlyoutState> {
     return (
       <Fragment>
         {legacyFileWarning && (
-          <span>
+          <>
             <EuiSpacer size="s" />
             {legacyFileWarning}
-          </span>
+          </>
         )}
         {indexPatternConflictsWarning && (
-          <span>
+          <>
             <EuiSpacer size="s" />
             {indexPatternConflictsWarning}
-          </span>
+          </>
```

Comment:
`<span></span>` tags are redundant and can be replaced with `<></>`

Reply:
+1

Reply:
I reduced the number of `Fragment`, also I unintentionally overwrote @joshuarrrr merge commit, I apologize 😨